src/common/mockFactory.ts

```diff
@@ -56,7 +65,7 @@ export default class MockFactory {
             id: `project-${name}`,
             name: `Project ${name}`,
             assets: {},
-            exportFormat: null,
+            exportFormat: MockFactory.exportFormat(),
```

Comment:
More of a question than anything. I've seen people still use `this` while using another static function from the same class. Which way is best?

Reply:
Technically it shouldn't work since there is no instance of the class for this to work against.  Since all these methods are static we should use it directly off the `MockFactory` like in this example.